src/ui/components/ui/UsdAmount/index.tsx

```diff
@@ -8,18 +12,22 @@ interface Props {
 }
 
 export function UsdAmount({ id, tokens, className }: Props) {
-  const usdPrices = usePopupSelector(state => state.usdPrices);
-
   const currentNetwork = usePopupSelector(state => state.currentNetwork);
   const isMainnet = currentNetwork === 'mainnet';
 
-  if (!usdPrices || !isMainnet) {
+  const usdPrices = useUsdPrices(useMemo(() => [id], [id]));
+
+  if (typeof usdPrices[id] !== 'string') {
+    return <Loader />;
+  }
+
+  if (!isMainnet || !usdPrices[id] || usdPrices[id] === '1') {
```

Comment:
You already check for `usdPrices[id]` above. Checking that it's a string is enough. If it's empty, then it's a backend problem. And we don't need a special case for '1' value, we decided to show usd prices for all assets always.